internal/jobqueue/jobqueuetest/jobqueuetest.go

```diff
@@ -404,6 +405,48 @@ func testCancel(t *testing.T, q jobqueue.JobQueue) {
 	require.NoError(t, err)
 }
 
+func testRequeue(t *testing.T, q jobqueue.JobQueue) {
+	// Cancel a non-existing job
```

Comment:
I think you forgot to change this comment.

Reply:
Fixed! Thanks!